src/Wrapper/WrapperBase.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace KrakenCore.Wrapper
+{
+    public class WrapperBase
+    {
+        protected readonly KrakenWrapper wrapper;
```

Comment:
Should adhere to either of the following rules:
- Non-private fields should be props instead
- Non-public fields should be prefixed with `_`